src/cpl/share_esmf/lnd_set_decomp_and_domain.F90

```diff
@@ -770,15 +802,32 @@ subroutine lnd_set_read_write_landmask(write_file, read_file, lndmask_glob, lndf
     ! local variables
     type(file_desc_t) :: pioid ! netcdf file id
     integer           :: dimid
-    character(len=CL) :: flandfrac = 'landfrac.nc'
+    integer           :: iun
+    integer           :: ioe
+    integer           :: ier  
+    logical           :: lexists
     !-------------------------------------------------------------------------------
 
+    ! TODO: remove status file before writing (unit 9876)
```

Comment:
- [ ] It looks to me like you have done what you wanted to do here. Do you agree? (i.e., should this TODO be removed?)

Reply:
@billsacks - thank you for checking. Yes - this is no longer needed. Thanks so much for reviewing the code.